guides/source/active_record_basics.md

```diff
@@ -606,6 +607,16 @@ irb> user.save!
 ActiveRecord::RecordInvalid: Validation failed: Name can't be blank
 ```
 
+The `create` method always returns an ActiveRecord object, regardless of
```

Comment:
```suggestion
The `create` method always returns an object, regardless of
```

Reply:
In Ruby everything is a object 😄, so this might be confusing (it won't return an instance of the `FalseClass`).
Maybe:
```suggestion
The `create` method always returns the record, regardless of
```

Reply:
`record` sounds more generic. I'm changing it to the `the record`

Reply:
@p8 Changes are done. Please review.